pages/mapTest.js

```diff
@@ -0,0 +1,173 @@
+import { connectToDatabase } from "../util/mongodb";
+import React from "react";
+
+//import { GoogleMap, Marker } from "react-google-maps";
+import {GoogleMap, Marker, InfoWindow, useLoadScript, } from "@react-google-maps/api";
+
+const { GOOGLE_MAPS_API_KEY } = process.env;
+
+const ObjectId = require('mongodb').ObjectID;
+
+const libraries = ["places"]; //libraries to avoid rerender?
+const mapContainerStyle = {
+	width: "100vw",
+	height: "100vh",
+};
+const center = {
+	lat: 40.7128,
+	lng: -74.0060,
+};
+
+const options = {
+	
+}
+
+export default function mApp({current_vendor, vendors, customers}){
+	
+	//console.log({current_vendor, vendors, customers});
+	//console.log("Hey");
+	//console.log(process.env.GOOGLE_MAPS_API_KEY);
+	
+	//NOTE BAD PRACTICE TO SHOEHORN API KEY IN USE ENVLOCAL MAYBE
+	
+	const {isLoaded, loadError} = useLoadScript({
+		googleMapsApiKey: "AIzaSyClhKv-XaZs679aVBkHB2dqTsQ1asckVx4",
+		libraries,
+	});
+	const [markers, setMarkers] = React.useState([]); //MAYBE A BETTER WAY TO DO THIS
+	
+	const [selected, setSelected] = React.useState(null); //FOR INFO BOX
+	
+	
+	//I think this is the right notation for function declaration
+	const sendLatLon = async (current_vendor, latitude, longitude) => {
```

Comment:
The code looks good from looking at it. I haven't had a chance to test it, so maybe we can review it tomorrow, Friday. I would say you have way too many extra blank lines, so if you could remove them, that would be better. In addition, comment out any console logs.